pkg/executor/join/base_join_probe.go

```diff
@@ -149,6 +149,11 @@ type baseJoinProbe struct {
 	spilledIdx []int
 
 	probeCollision uint64
+
+	memoryUsagePerRowBuffer []int64
```

Comment:
it looks to me we can reuse `serializedKeysLens` so there is no need to have `memoryUsagePerRowBuffer`?

Reply:
> it looks to me we can reuse `serializedKeysLens` so there is no need to have `memoryUsagePerRowBuffer`?

done